calcDMAS.js

```diff
@@ -0,0 +1,38 @@
+function calculateByDMAS(oper, operators, elements){
+    while(operators.includes(oper)){
+        for (i = 0; i < operators.length; i++){
+            if (operators[i] === oper){
```

Comment:
You can use switch here instead of if/else

Reply:
Fixed